### PR TITLE
DUMMY pr to test mpifileutils update

### PR DESCRIPTION
**Pull request by daltonbohning**
PR-repos-el7: mpifileutils-pkg@PR-10
PR-repos-leap15: mpifileutils-pkg@PR-10

Quick-Functional: true
Test-tag: datamover

Signed-off-by: Dalton Bohning <daltonx.bohning@intel.com>

### CODE REVIEW

File: src/tests/ftest/datamover/dummy.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+dummy file so git reports a diff so tests will actually run
```
Comment (brianjmurrell) [inline src/tests/ftest/datamover/dummy.txt:1]: FWIW, you can avoid this sort of thing by adding `Doc-only: false` to your commit message.

Comment (daltonbohning) [reply src/tests/ftest/datamover/dummy.txt:1]: I thought Doc-only skipped the test stage. Is this not true?

Comment (brianjmurrell) [reply src/tests/ftest/datamover/dummy.txt:1]: > I thought Doc-only skipped the test stage. Is this not true?

`Doc-only: true` does.  `Doc-only: false` disables it.

Comment (daltonbohning) [reply src/tests/ftest/datamover/dummy.txt:1]: Oh, I see. Doc-only is auto-detected :)
Thanks!

Comment (brianjmurrell) [reply src/tests/ftest/datamover/dummy.txt:1]: It is, for some _rough_ (surely not complete) value of auto-detected.

**Pull request closed without merging**